web-sdk/v3.md

```diff
@@ -119,72 +119,86 @@ const label = {
-    </div>
-    <div>
-        <dt>
-            <span><code>metadata</code></span>
-        </dt>
-        <dd>name of the global metadata object</dd>
-    </div>
-    <div>
-        <dt>
-            <span><code>closeOnSuccess</code></span>
-            <em>default: true</em>
-        </dt>
-        <dd>toggles the web form auto-close behaviour on successful card linking</dd>
-    </div>
-    <div>
-        <dt>
-            <span><code>schemes</code></span>
-            <em>object with boolean properties for mastercard, visa and amex. By default, all three are true.</em>
-        </dt>
-        <dd>toggles the availability of the Mastercard, Visa and Amex card schemes in the form</dd>
-    </div>
-    <div>
-        <dt>
-            <span><code>custom</code></span>
-        </dt>
-        <dd>object that allows you to customize the UI for the Web SDK. You can find the structure for it in the customization section below.</dd>
-    </div>
-</dl>
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td><code>companyName</code> (required)</td>
+    <td>string</td>
+    <td>The name of the company using card linking.</td>
+  </tr>
+  <tr>
+    <td><code>sdkKey</code> (required)</td>
+    <td>string</td>
+    <td>A valid SDK Key. You can find yours on the <a href="https://dashboard.fidel.uk/account/plan">Dashboard</a>. It starts with `pk_`.</td>
+  </tr>
+  <tr>
+    <td><code>programId</code> (required)</td>
+    <td>string</td>
+    <td>The id of the Fidel API Program to link the card to.</td>
+  </tr>
+  <tr>
+    <td><code>programName</code></td>
+    <td>string</td>
+    <td>The name of the Fidel API Program to link the card to.</td>
+  </tr>
+  <tr>
+    <td><code>programType</code></td>
+    <td>string<br /><em>Possible values: 'select-transaction', 'transaction-select', 'transaction-stream'</em></td>
+    <td>The type of the Fidel API Program to link the card to.</td>
+  </tr>
+  <tr>
+    <td><code>callback</code></td>
+    <td>function</td>
+    <td>Function to be called when the form is submitted.</td>
+  </tr>
+  <tr>
+    <td><code>container</code></td>
+    <td>HTML element</td>
+    <td>A wrapper element that you can wrap around the iframe.</td>
+  </tr>
+  <tr>
+    <td><code>countries</code></td>
+    <td>array of strings<br /><em>Possible values: 'GBR', 'IRL', 'USA', 'SWE'</em></td>
+    <td>If the array has only one value, that will be set as the country of issue for all collected cards and the country select box is removed from the UI.<br />If the array has multiple values, those will be the available options in the country select box.</td>
+  </tr>
+  <tr>
+    <td><code>language</code></td>
+    <td>string<br /><em>Possible values: 'EN', 'EN-BG', 'FR', 'JA', 'SV'</em><br />Default value: 'EN'</td>
+    <td>The language to be displayed. It follows `window.navigator.language` language codes.</td>
```

Comment:
I remember we passed this one to Yuta right? I think it's simple, and developers will understand what it means :+1: